integration-tests/api/__tests__/store/products/ff-product-categories.ts

```diff
@@ -151,6 +151,27 @@ describe("/store/products", () => {
       ])
     })
 
+    it("should return a list of products queried by category_id and q", async () => {
+      const api = useApi()
+      const productName = "Test product"
+      const params = `category_id[]=${categoryWithProductId}&q=${productName}&expand=categories`
```

Comment:
question: Does this category contain more than one product?

Reply:
Good point, added one with multiple products